### ISSUE TITLE
cargo-deb fails trying to strip unknown file formats

### ISSUE THREAD

**Issue by Razican** [mmstick/cargo-deb#67]
Hello,

I'm running cargo-deb in the [SUPERAndroidAnalyzer/super][super_repo] repository, and I get the following error with three files, all of them command line completions:

```
strip:/home/razican/workspace/super/super/target/release/super.bash: File format not recognized
cargo-deb: Command strip failed to launch
  because: exit code: 1
```

The affected files are the following:
 - `super.bash`
 - `super.fish`
 - `_super`

[super_repo]: https://github.com/SUPERAndroidAnalyzer/super

**Comment by kornelski** [comment]
How did this file get there?

**Comment by Razican** [comment]
They get generated in the `build.rs` build script. I guess it's good practice to generate the completions in the target directory, right?

**Comment by kornelski** [comment]
Indeed. Hmm, so cargo-deb needs a good way to distinguish between executables and other executable assets put in the target directory.

I could limit stripping only to native rust targets. Then stripping of everything else specified in package.metadata.deb.assets would be left up to build scripts...

**Comment by Razican** [comment]
That would be great, yes.

**Comment by Razican** [comment]
Hi @kornelski! Is there any news on this front?

**Comment by kornelski** [comment]
I haven't got time to implement this. Could you have a look yourself?

There's alredy built_binaries() method that should be plugged in.

**Comment by Razican** [comment]
OK, I will try to give it a look when I get some free time.

**Comment by Razican** [comment]
Hi, I just tried this again and it seems fixed. Is this the case?

**Comment by Razican** [comment]
Ping @kornelski

**Comment by kornelski** [comment]
No change since https://github.com/mmstick/cargo-deb/issues/67#issuecomment-392337187
